Fix IEEE S&P to cover alternate booktitles in some years

gannimo opened the pull request:
In some years (e.g., 2002, 2006, 2007) IEEE used "S&P" as booktitle for the Oakland proceedings. This seems to be a recent regression. With this additional booktitle, several missing years are picked up and parsed/included correctly.

gannimo committed:
Fix IEEE S&P to cover alternate booktitles in some years

In some years (e.g., 2002, 2006, 2007) IEEE used "S&P" as booktitle for the Oakland proceedings. This seems to be a recent regression.

emeryberger commented:
Thanks for the PR. I believe this needs a line in `util/csrankings.py` as well to handle the `S&P` case?

gannimo committed:
Added additional IEEE S&P name

Added additional name for IEEE Symposium on Security and Privacy based on Emery's feedback on the PR

gannimo commented:
You are absolutely right @emeryberger . Added the check to `util/csrankings.py` as well.
